doc/metrics/source-lines-of-code.md

```diff
@@ -0,0 +1,26 @@
+# Source lines of Code
+
+**Source lines of Code** approximate number of source code lines in a function or a method. Blank or comment lines are not counted.
```

Comment:
**Source lines of Code** is an approximate number of source code lines in a function or method. Blank or comment lines are not counted.

Reply:
fixed